lib/number-checks.js

```diff
@@ -0,0 +1,15 @@
+// To support Node v.0.10.x
+var number = module.exports = {
+  MAX_SAFE_INTEGER: Number.MAX_SAFE_INTEGER || 9007199254740991,
```

Comment:
Just a suggestion; wouldn't you thing the following would be neater and makes sense to the reader:
```
MAX_SAFE_INTEGER: Number.MAX_SAFE_INTEGER || Math.pow(2, 53) - 1,
```